grafana-plugin/package.json

```diff
@@ -40,25 +40,7 @@
   "author": "Grafana Labs",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@babel/plugin-proposal-class-properties": "^7.18.6",
-    "@babel/plugin-proposal-decorators": "^7.20.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
-    "@babel/plugin-proposal-object-rest-spread": "^7.18.9",
-    "@babel/plugin-proposal-optional-chaining": "^7.18.9",
-    "@babel/plugin-syntax-decorators": "^7.18.6",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/plugin-transform-destructuring": "^7.20.0",
-    "@babel/plugin-transform-react-constant-elements": "^7.18.12",
-    "@babel/plugin-transform-runtime": "^7.19.6",
-    "@babel/plugin-transform-typescript": "^7.18.12",
-    "@babel/preset-env": "^7.18.10",
-    "@babel/preset-react": "^7.18.6",
-    "@babel/preset-typescript": "^7.18.6",
```

Comment:
why the move to `dependencies`?

Reply:
Because they should be treated as dependencies and not dev dependencies. This is not related to thisPR but thought it will be good idea to do move them where they belong

Reply:
There's no difference in the way they are being used right now unless you pass special flags to yarn

Reply:
oh snap. I moved more than required - fixed now